src/Bundle/Tests/Functional/GridApiTest.php

```diff
@@ -229,6 +229,16 @@ public function it_filters_books_by_author_when_an_author_is_used_in_join_in_que
         $this->assertSame('A Study in Scarlet', $this->getFirstItemFromCurrentResponse()['title']);
     }
 
+    /** @test */
+    public function it_filters_books_by_multiple_state(): void
+    {
+        $this->client->request('GET', '/books/?criteria[state][]=unpublished&criteria[state][]=initial');
+
+        $this->assertCount(2, $result = $this->getItemsFromCurrentResponse());
+        $this->assertSame('A Study in Scarlet', $this->getFirstItemFromCurrentResponse()['title']);
+        $this->assertSame('The Lost World', end($result)['title']);
```

Comment:
We could use some helper method here as well
```suggestion
        $this->assertSame('The Lost World', $this->getLastItemFromCurrentResponse()['title']);
```

Reply:
This helper does not exist. Do you want I write this helper for one use?

Reply:
I meant the creation of the new one ;) But, don't worry, it's good to go for me. Probably after Sylius 1.7 release

Reply:
Ok, this bug fix will be apply only for the version 1.7 or older too ?